.github/workflows/lint.yaml

```
@@ -3,7 +3,7 @@ name: Lint
 on: [push, pull_request]
 
 jobs:
-  build:
+  lint:
     runs-on: ubuntu-latest
```

Disable creation of virtualenv

```suggestion
jobs:
  lint:
    runs-on: ubuntu-latest
    env:
      # Hide the graphical elements from pipenv's output
      POETRY_VIRTUALENVS_CREATE: false
      PIPENV_HIDE_EMOJIS: 1
      PIPENV_NOSPIN: 1
```